.gitignore

```diff
@@ -43,3 +43,4 @@ pybind11Targets.cmake
 /pybind11/share/*
 /docs/_build/*
 .ipynb_checkpoints/
+tests/main.cpp
```

Comment:
@Lalaland Is this main.cpp still relevant here?

Reply:
Yes, it's a build artifact that should always be ignored.